### DISCUSSION TITLE
/stats page data is empty

### DISCUSSION THREAD

**Discussion by jianvector** [jasonish/evebox#306] [General]
How to do it with agent, and send log data to server for display on stats pages?

Thanks.

**Comment by jasonish** [comment]
If you are already sending your log file to EveBox, the stats should just populate.  Do you have the `stats` enabled in your Suricata configuration?

Otherwise, please provide more detail like what dabatase you are using, how you're sending events, etc.

**Comment by jianvector** [comment]
oops! stats disable in Suricata configuration.

agent.yaml

server:
url: http://127.0.0.1:5636
input:
paths:
- "/var/log/suricata/eve.json"
- "/var/log/suricata/stats.log"

agent yaml config right?

Jason Ish ***@***.***> 于2024年4月11日周四 03:12写道：

**Comment by jasonish** [comment]
EveBox cannot consume the Suricata `stats.log`.

Instead, Suricata *includes* `stats` records in the `eve.json` by default. This is how EveBox gets its stats. It should all just work with a default configuration.  If you are seeing alerts in EveBox you should also see stats. Unless you disabled stats output in Suricata.
